tests/high_frequency_telemetry/utilities.py

```diff
@@ -0,0 +1,1260 @@
+        object_names: List of object names or comma-separated string
+        object_counters: List of counter names or comma-separated string
+    """
+    if isinstance(object_names, list):
+        object_names = ",".join(object_names)
+    if isinstance(object_counters, list):
+        object_counters = ",".join(object_counters)
+
+    group_cmd = (
+        f'redis-cli -n 4 HSET "HIGH_FREQUENCY_TELEMETRY_GROUP|'
+        f'{profile_name}|{group_name}" '
+        f'"object_names" "{object_names}" '
+        f'"object_counters" "{object_counters}"'
+    )
+
+    result = duthost.shell(group_cmd, module_ignore_errors=False)
+    logger.info(f"Created high frequency telemetry group '{group_name}' "
+                f"for profile '{profile_name}': "
+                f"objects={object_names}, counters={object_counters}")
+    return result
+
+
+def cleanup_hft_config(duthost, profile_name, group_names=None):
+    """
+    Clean up high frequency telemetry configuration.
+
+    Args:
+        duthost: DUT host object
+        profile_name: Name of the profile to clean up
+        group_names: List of group names to clean up (optional, if None,
+                     will clean all groups for the profile)
+    """
+    cleanup_commands = []
+
+    # Clean up profile
+    cleanup_commands.append(
+        f'redis-cli -n 4 DEL "HIGH_FREQUENCY_TELEMETRY_PROFILE|{profile_name}"'
+    )
+
+    # Clean up groups
+    if group_names:
+        if isinstance(group_names, str):
+            group_names = [group_names]
+        for group_name in group_names:
+            cleanup_commands.append(
+                f'redis-cli -n 4 DEL "HIGH_FREQUENCY_TELEMETRY_GROUP|'
+                f'{profile_name}|{group_name}"'
+            )
+    else:
+        # Clean up all groups for this profile (use pattern matching)
+        pattern_cmd = (
+            f'redis-cli -n 4 KEYS "HIGH_FREQUENCY_TELEMETRY_GROUP|'
+            f'{profile_name}|*"'
+        )
+        result = duthost.shell(pattern_cmd, module_ignore_errors=True)
+        if result['rc'] == 0 and result['stdout_lines']:
+            for key in result['stdout_lines']:
+                if key.strip():
+                    cleanup_commands.append(
+                        f'redis-cli -n 4 DEL "{key.strip()}"'
+                    )
+
+    # Execute cleanup commands
+    for cmd in cleanup_commands:
+        duthost.shell(cmd, module_ignore_errors=True)
+
+    logger.info(f"Cleaned up high frequency telemetry configuration "
+                f"for profile '{profile_name}'")
+
+
+def run_countersyncd_and_capture_output(duthost, timeout=120):
+    """
+    Run countersyncd command and capture output.
+
+    Args:
+        duthost: DUT host object
+        timeout: Timeout in seconds (default: 120)
+
+    Returns:
+        dict: Command result with stdout, stderr, rc
+    """
+    countersyncd_cmd = (
+        f'timeout {timeout} docker exec swss countersyncd -e '
+        f'--max-stats-per-report 0 '
+    )
```

Comment:
```suggestion
def run_countersyncd_and_capture_output(duthost, timeout=120, stats_interval=60):
...
        f'--max-stats-per-report 0 '
        f'--stats-interval {stats_interval}'
    )
```
I suggest adding the option the override the default stats-interval, from my check this is needed for the longer poll interval tests